src/OrbitQt/orbitmainwindow.cpp

```diff
@@ -729,9 +729,17 @@ void OrbitMainWindow::OnNewSelectionBottomUpView(
 }
 
 std::string OrbitMainWindow::OnGetSaveFileName(const std::string& extension) {
-  std::string filename =
-      QFileDialog::getSaveFileName(this, "Specify a file to save...", nullptr, extension.c_str())
-          .toStdString();
+  QFileDialog dialog(this);
+  dialog.setFileMode(QFileDialog::FileMode::AnyFile);
+  dialog.setAcceptMode(QFileDialog::AcceptMode::AcceptSave);
+  dialog.setNameFilter((extension + " (*" + extension + ")").c_str());
+  dialog.setWindowTitle("Specify a file to save...");
+  dialog.setDirectory(nullptr);
+  std::string filename;
+
+  if (dialog.exec() != 0 && dialog.selectedFiles().size() > 0) {
```

Comment:
Nit: `QDialog::Rejected` instead of `0` or `dialog.exec() == QDialog::Accepted` (as you prefer).